install-manager/src/main/java/cm/aptoide/pt/install_manager/DownloadInfo.kt

```diff
@@ -0,0 +1,6 @@
+package cm.aptoide.pt.install_manager
+
+data class DownloadInfo(
```

Comment:
Looks like it is not used anywhere else than PackageDownloader implementations. And Task uses only progress value from it. How about using it also in the `Task.State.Downloading` state?

Reply:
Let's create a tech debt ticket about this